packages/synapse-react-client/src/components/TableColumnSchemaEditor/ColumnModelFormFields/FieldWithRecommendedMinimum.tsx

```diff
@@ -0,0 +1,91 @@
+import {
+  alpha,
+  SxProps,
+  TextField,
+  TextFieldProps,
+  Theme,
+  Tooltip,
+} from '@mui/material'
+import { ColumnType, ColumnTypeEnum } from '@sage-bionetworks/synapse-types'
+import React from 'react'
+import {
+  getFormDataValueAsNumber,
+  isFormDataValueSmallerThanRecommendedValue,
+} from '../TableColumnSchemaEditorUtils'
+
+export type FieldWithRecommendedValueProps = Omit<TextFieldProps, 'value'> & {
+  value: string | number | undefined
+  originalValue?: string | number | undefined
+  recommendedValue?: number
+  columnType: ColumnType
+}
+
+export function FieldWithRecommendedMinimum(
+  props: FieldWithRecommendedValueProps,
+) {
+  const {
+    value,
+    originalValue,
+    recommendedValue,
+    columnType,
+    error: errorFromProps,
+    ...textFieldProps
+  } = props
+
+  const isSmallerThanRecommendedValue =
+    isFormDataValueSmallerThanRecommendedValue(value, recommendedValue)
+
+  const hasChangedFromOriginal =
+    originalValue != null &&
+    getFormDataValueAsNumber(value) !== getFormDataValueAsNumber(originalValue)
+
+  const showError = !!errorFromProps || isSmallerThanRecommendedValue
+
+  let tooltipText = null
```

Comment:
Error message for recommended size is now in a tooltip instead of below the input